### PR TITLE
Avoid choosing the MapTileSource in app preferences. Let Mapnik by default.

### PR DESCRIPTION
**Pull request by xsaco07**
This PR fix issue #96, #125, #127 

mapTileSources

The options `CycleMap` and `MapQuest` are not available any more for using them in the app.

The only source available that woks in the same way is MAPNIK so we decided that, by the moment, the app will use by default the MAPNIK tile source. If this is the best for the app and users, we can let this change be permanent.

With this PR the map tile source can not be changed. This is for avoiding exceptions and because MAPNIK will be used anyway if the selected source does not work correctly (throws an exception) and this will happen if `MapQuest` or `CycleMap` are chosen.

In conclusion the functionality of choosing the map tile source was useless right now.

### CODE REVIEW

**Comment by EmmanuelAB** [comment]
I checked this PR and indeed now the option to select the tile provider has been removed imagen
